apps/web/src/components/CareLocationSelector.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Dropdown } from './Dropdown';
+import { getCareLocations } from '../services/carelocations';
+
+interface UnitProps {
+  id: string;
+  unitName: string;
+}
+
+export const CareLocationSelector = () => {
+  const [careLocations, setCareLocations] = React.useState([]);
+
+  React.useEffect(() => {
+    (async () => {
+      setCareLocations(
+        (await getCareLocations()).map((unit: UnitProps) => ({ ...unit, value: unit.unitName })),
+      );
+    })();
+  }, []);
+
+  return (
+    <>
+      <Dropdown dropdownOptions={careLocations}></Dropdown>
```

Comment:
Also, only after selecting the radio btn, this drop down will appear. Kindly go through this design https://www.figma.com/file/QuGyUBu1JEkNaGTb8ZfC3l/Care-mapping-design?node-id=173%3A10180

Reply:
Right, good point! Are the radio buttons part of this Jira ticket or should it be a separate one?

Reply:
It's a part of this ticket

Reply:
Thank you, I'll go ahead and add it to the Jira ticket description too, seems to be missing as of now 👍